monome/teletype
# Pattern mode key bindings

#### What does this PR do?

Adds the following keybindings to pattern/tracker mode:

```
ALT-[ ]|SEMITONE UP, DOWN
CTRL-[ ]|FIFTH UP, DOWN
SH-[ ]|OCTAVE UP, DOWN
ALT-<0-9>|NUM SEMITONES UP
// 0 = 10 and 1 = 11
SH-ALT-<0-9>|NUM SEMITONES DOWN
// 0 = 10 and 1 = 11
```

Demo video: https://www.youtube.com/watch?v=gxOZY_MgsvE

#### Provide links to any related discussion on [lines](https://llllllll.co/).

https://llllllll.co/t/teletype-firmware-discussion/13961/41

#### How should this be manually tested?

1. Enter Pattern/Tracker Mode
2. Try key commands listed above
3. Verify values entered match values in N table

#### Any background context you want to provide?

#### If the related Github issues aren't referenced in your commits, please link to them here.

#### I have,


NOTE: `make format` formatted a bunch of lines in help_mode.c that were not in my original changeset
